index.js

```diff
@@ -143,6 +144,11 @@ module.exports = async (input = 'patch', options) => {
 		task: () => exec('git', ['push', '--follow-tags'])
 	});
 
+	tasks.add({
+		title: 'Releasing on Github',
+		task: () => release(options)
```

Comment:
This task needs `skip` logic if the repo isn't hosted on GitHub.

Reply:
I think using `np` could already safely assume you are using Github? It's just Github integration all around the places in `np`

Reply:
`np` added support for GitLab a few months ago, see #282.